packages/@aws-cdk/aws-scheduler-alpha/rosetta/default.ts-fixture

```diff
@@ -6,8 +6,10 @@ import * as iam from 'aws-cdk-lib/aws-iam';
 import * as kms from 'aws-cdk-lib/aws-kms';
 import * as sqs from 'aws-cdk-lib/aws-sqs';
 import * as cloudwatch from 'aws-cdk-lib/aws-cloudwatch';
+// This import will fail as aws-scheduler-alpha does not depend on aws-scheduler-targets-alpha to avoid cyclic dependency 
+// import * as targets from '@aws-cdk/aws-scheduler-targets-alpha';
```

Comment:
Calling this out explicitly in comments too @kaizencc. Do you have any ideas for how to handle that?

Reply:
in package.json we can add 

```json
  "jsiiRosetta": {
    "exampleDependencies": {
      "@aws-cdk/aws-schduler-targets-alpha": "^2.92.0",
    }
  },
```

Reply:
Thanks! That worked.